src/templates/Event.js

```diff
@@ -1,23 +1,98 @@
 import React, { Component } from 'react'
+import styled, { css } from 'styled-components'
 import PropTypes from 'prop-types'
 import ReactMarkdown from 'react-markdown'
 import { EventTagList, EventSchedule } from './events'
+import Helmet from 'react-helmet'
+import { media } from '../theme/media'
 
-export default class Event extends Component {
-  state = { event: this.props.data.contentfulEvent }
+const PageWrapper = styled.div`
+  position: relative;
+  margin: 0 auto;
+  max-width: ${props => props.theme.breakpoints[3]};
+  background-color: white;
+`
 
+const Title = styled.h1`
+  color: ${props => props.theme.colors.indigo};
+`
+
+const HeroImageAndTitle = styled.div`
+  display: flex;
+  flex-direction: column-reverse;
+  ${media.desktop`
+    flex-direction: column;
+  `};
+`
+
+const ContentWrapper = styled.div`
+  padding: 30px 20px;
+  width: 100vw;
+  ${media.desktop`
```

Comment:
this is a fairly common styled-components pattern. A bit easier to read than `@media (min-width: ${props => props.theme.breakpoints[3]}) { ... }`